### ISSUE TITLE
update cost related config

### ISSUE THREAD

**Issue by dabasov** [0chain/0chain#1063]
Cost related settings in the sc.yaml should be updatable the same way the other settings are updatable. Implement for smart contracts:
- faucetsc
- interestpoolsc
- minersc
- storagesc
- vestingsc
- zcnsc

Cost section example
```
    cost:
      add_miner: 100
      add_sharder: 100
      delete_miner: 100
      delete_sharder: 100
      miner_health_check: 100
      sharder_health_check: 100
      contributeMpk: 100
      shareSignsOrShares: 100
      wait: 100
      update_globals: 100
      update_settings: 100
      update_miner_settings: 100
      update_sharder_settings: 100
      payFees: 100
      feesPaid: 100
      mintedTokens: 100
      addToDelegatePool: 100
      deleteFromDelegatePool: 100
      sharder_keep: 100
```

**Comment by ma2b0043** [comment]
@dabasov when can we expect to close this issue?

**Comment by iamrz1** [comment]
> @dabasov when can we expect to close this issue?

I'm working on a PR to address this. We will have it merged by the end of this coming week.

**Comment by peterlimg** [comment]
Closed via #1076
